### ISSUE TITLE
documentation: explain the "probability" label

### ISSUE THREAD

**Issue by vrooje** [zooniverse/aggregation#11]
Just a note for the aggregation documentation - it will be important to spend some time describing in as close to layperson's terms as possible what is meant by "probability" as used in the aggregation exports, including caveats.

**Comment by ggdhines-zz** [comment]
definitely - I'm just not sure where to include such documentation. Suggestions welcome

**Comment by vrooje** [comment]
We just drafted other docs in Google Drive and shared widely, before deciding on a final version to go on the site.

**Comment by ggdhines-zz** [comment]
yeah, we need a place on the site for this sort of stuff to go. Will probably become clearer as we integrate aggregation stuff into the site. I am going to write some stuff up and send it around.

**Comment by chrislintott** [comment]
Yep, use Google Docs for now and we’ll worry about the rest later.
